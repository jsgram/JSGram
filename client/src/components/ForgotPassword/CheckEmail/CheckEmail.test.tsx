import { CheckEmail } from './index';
import { shallow } from 'enzyme';
import configureStore from 'redux-mock-store';
import React from 'react';
import { Provider } from 'react-redux';

describe('CheckEmail component:', () => {
    let renderer: any;
    const checkEmail = jest.fn(() => 'some value');
    const handleSubmit = jest.fn(() => 'some value');
    const submitting = jest.fn(() => 'some value');

    beforeEach(() => {
        const store = configureStore()();
        renderer = shallow(<Provider store={store}>
                               <CheckEmail
                                   checkEmail={checkEmail}
                                   handleSubmit={handleSubmit}
                                   submitting={submitting}
                               />
                           </Provider>);
    });

    test('render - success', () => {
        expect(renderer).toMatchSnapshot();
    });
});
