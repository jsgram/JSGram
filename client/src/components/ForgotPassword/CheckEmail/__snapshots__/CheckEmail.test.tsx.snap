// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckEmail component: render - success 1`] = `ShallowWrapper {}`;
